### PR TITLE
Update Boilerplates List

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?

- resolves https://github.com/eoscostarica/full-stack-boilerplate/issues/41

### Steps to test
1. Navigate to the tutorials section
2. See Run the boilerplate section on the sidebar

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/tutorials/run-boilerplate.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# Running the Boilerplate
+
+## Getting Started
+
+Some things you need before getting started:
+
+- [git](https://git-scm.com/)
+- [node.js](https://nodejs.org/es/)
+- [docker](https://www.docker.com/)
+- [docker-compose](https://docs.docker.com/compose/)
```
Comment (JAMares) [inline docs/tutorials/run-boilerplate.md:18]: see my comments in Spanish

**Review by JAMares — CHANGES_REQUESTED** [review]

**Review by xavier506 — CHANGES_REQUESTED** [review]
# Lets re arrange this list a bit :

## EOS Boilerplates 

- [Full Stack Boilerplate](https://github.com/eoscostarica/full-stack-boilerplate)
- [Front End](https://github.com/eoscostarica/backend-boilerplate)
- [Back End](https://github.com/eoscostarica/backend-boilerplate)

## WAX Boilerplates 

- [WAX WebApp](https://github.com/edenia/wax-webapp-boilerplate)
- [Wax Full Stack Boilerplate](https://github.com/edenia/wax-full-stack-boilerplate)

## PROTON Boilerplate 

- [Proton Full Stack Boilerplate](https://github.com/edenia/proton-full-stack-boilerplate)

## General Boilerplates 

- [WebApp Boilerplate](https://github.com/eoscostarica/webapp-boilerplate)
- [Hapi Boilerplate](https://github.com/eoscostarica/hapi-boilerplate)
- [Hasura Boilerplate](https://github.com/eoscostarica/hasura-boilerplate)

File: docs/boilerplate.md [hunk 2]
```diff
@@ -10,6 +10,25 @@ A boilerplate is a framework or template for the development of a project. It is
...
+- [WebApp Boilerplate](https://github.com/eoscostarica/webapp-boilerplate)
+- [Hapi Boilerplate](https://github.com/eoscostarica/hapi-boilerplate)
+- [Hasura Boilerplate](https://github.com/eoscostarica/hasura-boilerplate)
+
+## WAX Boilerplates 
+
+- [WAX WebApp](https://github.com/edenia/wax-webapp-boilerplate)
+- [Wax Full Stack Boilerplate](https://github.com/edenia/wax-full-stack-boilerplate)
+
+## PROTON Boilerplate
```
Comment (xavier506) [inline docs/boilerplate.md:27]: No need for all CAPS here

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by warnerHurtado — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**